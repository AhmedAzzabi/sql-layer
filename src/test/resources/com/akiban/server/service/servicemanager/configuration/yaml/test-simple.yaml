- bind:
    my_one : alpha
    my_two : beta
- bind:
    my_three : gamma
- lock: delta
- require : epsilon
- require locked: zeta
- locked : eta
- bound : theta