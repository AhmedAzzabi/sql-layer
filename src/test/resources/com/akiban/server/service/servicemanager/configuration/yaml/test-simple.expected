BIND my_one -> alpha
BIND my_two -> beta
BIND my_three -> gamma
LOCK delta
REQUIRE epsilon
REQUIRE zeta
MUST BE LOCKED zeta
MUST BE LOCKED eta
MUST BE BOUND theta
SECTION END